tests/components/teslemetry/test_sensor.py

```diff
@@ -25,3 +30,26 @@ async def test_sensors(
     entry = await setup_platform(hass, [Platform.SENSOR])
 
     assert_entities(hass, entry.entry_id, entity_registry, snapshot)
+
+    # Coordinator refresh
+    freezer.tick(timedelta(seconds=SYNC_INTERVAL))
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done()
```

Comment:
Why do we need this when the test is over?

Reply:
I believe this is the only test that covers the second coordinator refresh

Reply:
Removed

Reply:
It's not bad if we have a test that actually change something in the data, move time and assert it did change so would be happy to see that included here (don't need snapshot, can just check a single entity perhaps) 👍

Reply:
It didnt impact coverage so I think it was there to test something that was not included in this PR.

Reply:
Such test would not impact coverage as updating is done on first refresh so therefore coverage is already there.
Sometimes however it's good to include tests to see that something works regardless coverage.

Reply:
I have re-added the test with a single state assertion.